README.md

```diff
@@ -1 +1,62 @@
-# webpack
\ No newline at end of file
+# FreeEpicGames
+A project that display all the free weekly games provided by epic games, We have used Epic Games API provided by Rapid API.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+[Live Demo Link]()
```

Comment:
### [OPTIONAL]
The link is not working. Kindly add the valid and updated link 👍🏻